### PR TITLE
Add "SR-BH 2020" dataset

### PR DESCRIPTION
**Pull request by Maspital**
Do not merge before #70

### CODE REVIEW

File: content/datasets/sr_bh_2020.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+| **User Emulation**       | Real users                                 |
+|                          |                                            |
+| **Packed Size**          | -                                          |
+| **Unpacked Size**        | 436 MB                                     |
+| **Download Link**        | [goto](https://doi.org/10.7910/DVN/OGOIXX) |
+
+***
+
+### Overview
+The SR-BH 2020 dataset (meaning of acronym unknown) consists of twelve days of traffic collected from a honeypot machine running a Wordpress webserver.
```
Comment (ru37z) [inline content/datasets/sr_bh_2020.md:35]: I suppose it's the surnames of the first two authors

File: content/datasets/sr_bh_2020.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+Each individual request is recorded using a small set of fields;
+the distinguishing feature of the dataset is the inclusion of multi-class labels, where each request is either labeled "normal" or assigned to at least one of twelve MITRE CAPEC attack classifications.
+This dataset is then used by the authors to showcase several machine learning applications, whereby textual features are translated into numerical ones by taking the mean ASCII-value of all characters.
+
+It should be noted that some features within the dataset are strongly biased towards one side, for example `http_status_message`, which will always be "Not Found" for *all* attacks (although some benign requests have this value as well).
+This is not addressed by the authors, though I do assume that their main focus was on correctly assigning different attack labels, where this is less of a factor compared to pure attack detection (as the model still has to figure out the correct set of attack labels).
+
+### Environment
+The honeypot consists of a Wordpress webserver running on a virtual machine that is exposed to the internet.
```
Comment (ru37z) [inline content/datasets/sr_bh_2020.md:44]: Internet should be capitalized when "the" Internet is meant

File: content/datasets/sr_bh_2020.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+A separate router is configured between VM and internet, which causes all requests received by the webserver to originate from the local IP of this router (for the sake of anonymization).
+
+The Web Application Firewall (WAF) [ModSecurity](https://github.com/owasp-modsecurity/ModSecurity), which analyzes traffic at the application layer using a set of rules (Core Rule Set 3.3.0), is installed in "Detection Mode", meaning it does not block any requests and instead only generates logs.
+These logs, which inform about individual request and already contain CAPEC labels assigned by ModSecurity, are collected, and the virtual machine is reset every day.
+
+### Activity
+As this is a honeypot, there is no pre-planned malicious activity.
+Benign behavior is also not detailed in any fashion, even though it makes up a notable portion of request (~58%).
```
Comment (ru37z) [inline content/datasets/sr_bh_2020.md:52]: request -> requests

File: content/datasets/sr_bh_2020.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+- 88 - OS Command Injection
+- 126 - Path Traversal
+- 66 - SQL Injection (these are the majority of attacks)
+- 16 - Dictionary-based Password Attack
+- 310 - Scanning for vulnerable software
+- 153 - Input Data Manipulation
+- 274 - HTTP Verb Tampering
+- 194 - Fake the source of data
+- 34 - HTTP Response Splitting
+- 33 - HTTP Request Smuggling
```
Comment (ru37z) [inline content/datasets/sr_bh_2020.md:67]: Inconsistent capitalization in this listing

**Review by ru37z — CHANGES_REQUESTED** [review]

File: content/datasets/sr_bh_2020.md (hunk 1 above)
Comment (Maspital) [reply content/datasets/sr_bh_2020.md:35]: Should I add this assumption here or just leave it out entirely?

Comment (Maspital) [reply content/datasets/sr_bh_2020.md:35]: eh, I'll just add it, why not

**Pull request merged**